src/rpdk/core/templates/resource-role.yml

```diff
@@ -7,7 +7,7 @@ Resources:
   ExecutionRole:
     Type: AWS::IAM::Role
     Properties:
-      MaxSessionDuration: 8400
+      MaxSessionDuration: 43200
```

Comment:
couldn't this be templated from the value in the schema?

Reply:
will try that out